src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/temporaryabsences/TemporaryAbsenceService.kt

```diff
@@ -36,11 +36,13 @@ class TemporaryAbsenceService(
 
   fun confirmTemporaryAbsencesArrival(
     prisonNumber: String,
-    request: ConfirmTemporaryAbsenceRequest
+    request: ConfirmTemporaryAbsenceRequest,
+    movementId: String? = null
```

Comment:
can we call this arrival ID? 

movement ID is how the BASM domain refers to it, arrival ID is WPIP. 
I was looking through the code base and its a bit inconsistent, maybe it would be worth having a go at making it more consistent in a follow up or raising a tech debt ticket for it?

Reply:
I did start it off as arrivalId, but then realised that it was stored as movementId in the DB so changed it to that. As you say, we probably should be referring to it as arrivalId in the DB if that's our term for the data.

I'll change it to arrivalId and update the description to say something along the lines of this being the BASM movementId.

I'll create a ticket as well to go through the code and DB and update our references to arrivalId 😄